scripts/install_and_test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+cd $1/jstest/sdks
+
+for D in ./*; do
+    if [ -d "$D" ]; then
+        cd "$D"
+        npm i && npx playwright install \
+          && npm install @yext/pages $1/packages/pages/latest/* \
```

Comment:
Re-commenting this: Is playwright install necessary? I'd think npm i would cover it if the tests in the folder require it as it'd be an npm dep.

Reply:
Playwright install is separate from node package.json and doesn't get included in `npm i`. `Npm i` does install playwright, but the playwright install command installs the browsers and has to be run separately.